src/makeCAF.C

```diff
@@ -486,14 +496,23 @@ void loop(CAF &caf,
       cafmaker::LOG_S("loop()").INFO() << "Global trigger idx : " << ii << ", reco filler: '" << fillerTrigPair.first->GetName() << "', reco trigger eventID: " << fillerTrigPair.second.evtID << "\n";
       fillerTrigPair.first->FillRecoBranches(fillerTrigPair.second, caf.sr, par, &truthMatcher);
     }
+
+    // Once all the reco fillers have been called, let's call the matching fillers
+    for (const std::unique_ptr<cafmaker::IRecoBranchFiller>& filler : recoFillers)
+    {
+      if (filler->GetName() == "LArTMSMatcher" || filler->GetName() == "LArMINERvAMatcher")
```

Comment:
and this one would become

```suggestion
      if (filler->FillerType == RecoFillerType::Matcher)
```

Reply:
Implemented